src/main/webapp/i18n/de/exercise.json

```diff
@@ -62,6 +62,8 @@
             "submissionCount": "Abgaben",
             "score": "Punktzahl",
             "duration": "Dauer",
+            "showBuildFailed": "Nur \"Build Failed\" anzeigen",
```

Comment:
In English translation you have it without `"`. Can you please use it with or without `"` in both cases? In my opinion you don't need it

Reply:
True, without quotes looks fine as well.